src/components/Toolbar/css/Popup.css

```diff
@@ -0,0 +1,52 @@
+.popup-menu {
+  font-size: 14px;
+}
+
+.popup-menu--button:hover {
+  background-color: white;
+  color: #000;
+}
+
+.popup-menu--content {
+  position: absolute;
+  top: 100%;
+  background-color: var(--main-gray-3);
+  padding: 20px;
+  display: none;
+  z-index: 12;
+  width: 33%;
+  color: #000;
+}
+
+.popup-menu--content ul {
+  padding-left: inherit;
+}
+
+.popup-menu--content li {
+  padding-bottom: 15px;
+}
+
+.popup-menu--content li:last-child {
+  padding-bottom: 0px;
+}
+
+.active .popup-menu--content {
+  display: block;
+}
+
+.popup-menu--content--popup:first-child {
+  border-top: 1px solid var(--main-gray-2);
+}
+
+.popup-menu--content--popup {
+  text-align: left;
+  padding: 15px;
+  border-left: 1px solid var(--main-gray-2);
+  border-bottom: 1px solid var(--main-gray-2);
+  border-right: 1px solid var(--main-gray-2);
+  cursor: pointer;
```

Comment:
Not sure if cursor: pointer is intentional, but most non-clickable places in the app don't have this

Reply:
oh nope, great catch. I only wanted the `cursor: pointer` for the button itself. I'll remove this!